tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py39-lint, py3{8,9,10,11,12}-{readme,unit,end_to_end,numerical,minimum,tutorials}
+envlist = py39-lint, py3{8,9,10,11,12,13}-{readme,unit,end_to_end,numerical,minimum,tutorials}
 
 [testenv]
 skipsdist = false
```

Comment:
We can delete `tox.ini` i think we are not using tox anymore.

Reply:
Maybe that could be a separate issue since we mention tox in the makefile, contributing documentation, etc.